asf/sam0/include/samc21/component-version.h

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
The commit brief should be no more than 65 characters.
You can provide a better description in the commit body if necessary.
Please split in 2 commits:
 * one for introduce C21 ASF
 * another to add bindings.

Reply:
@nandojve can I make 2 commits in one PR?

Reply:
Yes, you can make more than 1 commit per PR.